app/client/templates/BoardgamesVote.tsx

```diff
@@ -0,0 +1,61 @@
+import { Alert } from "@client/components/alert/Alert";
+import { BackToBoardgamesLink } from "@client/components/boardgames/BackToBoardgamesLink";
+import { CenteredContentContainer } from "@client/components/containers/CenteredContentContainer";
+import { Layout } from "@client/components/Layout";
+import { PageTitle } from "@client/components/PageTitle";
+import { CSRFToken, reverse, templates } from "@reactivated";
+import React, { useState } from "react";
+
+export const Template = (props: templates.BoardgamesVote) => {
+  const [votes, setVotes] = useState(props.boardgame_ids_vote_for);
+
+  const onCheckboxChange = (boardGameId: number, checked: boolean) => {
+    if (checked) {
+      setVotes([...votes, boardGameId]);
```

Comment:
Theoretically, it would be better to use functional setState in both cases, it may cause less bugs
```suggestion
      setVotes(oldVotes => ([...oldVotes, boardGameId]));
```

Reply:
Done